.gitignore

```diff
@@ -13,7 +14,16 @@ logs
 /jsettlers.common/resources/images/*
 !/jsettlers.common/resources/images/movables.txt
 /jsettlers.common/resources/save
+config.prp
 
 /jsettlers.main.swing/*.log
 
-/jsettlers.tests/out
+#IDE Files
```

Comment:
In an ideal world everyone would not commit their IDE specific files. Realistically it's much easier to prevent it from happening by adding the ignores, given as you say that eclipse is going to be the most likely one used.